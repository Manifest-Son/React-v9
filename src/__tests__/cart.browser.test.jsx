import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import Cart from "../Cart";

test("Snapshot with nothing in cart", () => {
  const { asFragment } = render(<Cart cart={[]} />);
  expect(asFragment()).toMatchInlineSnapshot(`
      <DocumentFragment>
        <div
          class="cart"
        >
          <h2>
            Cart
          </h2>
          <ul />
          <p>
            Total: KESÂ 0.00
          </p>
          <button>
            Checkout
          </button>
        </div>
      </DocumentFragment>
    `);
});

test("Lol", () => {
  expect({
    id: "calabrese",
    name: "The Calabrese Pizza",
    category: "Supreme",
    description: "lol pizza from Calabria",
    image: "/public/pizzas/calabrese.webp",
    size: { S: 12.25, M: 16.25, L: 20.25 },
  }).toMatchSnapshot();
});
