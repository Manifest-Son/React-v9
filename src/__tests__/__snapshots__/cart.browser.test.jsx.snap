// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Lol 1`] = `
{
  "category": "Supreme",
  "description": "lol pizza from Calabria",
  "id": "calabrese",
  "image": "/public/pizzas/calabrese.webp",
  "name": "The Calabrese Pizza",
  "size": {
    "L": 20.25,
    "M": 16.25,
    "S": 12.25,
  },
}
`;
